src/App.tsx

```diff
@@ -1,10 +1,70 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
+import cn from 'classnames';
+
 import { UserWarning } from './UserWarning';
+import { Status } from './types/Status';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList/TodoList';
+import { Footer } from './components/Footer/Footer';
 
-const USER_ID = 0;
+const USER_ID = 11098;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [hasError, setHasError] = useState(false);
+  const [filterStatus, setFilterStatus] = useState(Status.ALL);
+
+  useEffect(() => {
+    getTodos(USER_ID)
+      .then((todosfromserver) => {
```

Comment:
```suggestion
      .then((todosFromServer) => {
```
Don't forget about camelCase)